enterprise/views.py

```diff
@@ -365,6 +365,11 @@ def get_default_context(self, enterprise_customer, platform_name):
                 _('If you decline to consent, that fact may be shared with {enterprise_customer_name}.').format(
                     enterprise_customer_name=enterprise_customer.name
                 ),
+                _(
```

Comment:
Please ignore if I am wrong. I think you are only adding the value without any key. For reference please check L369.

Reply:
Actually, this is another item on the list. the main dict contains an item that is a list and this new message is part of that list.